addon/edit/matchbrackets.js

```diff
@@ -106,13 +121,15 @@
       cm.off("cursorActivity", doMatchBrackets);
     if (val) {
       cm.state.matchBrackets = typeof val == "object" ? val : {};
+      cm.state.matchBrackets.fatCursor = 
+          cm.getWrapperElement().classList.contains('cm-fat-cursor');
       cm.on("cursorActivity", doMatchBrackets);
     }
   });
 
   CodeMirror.defineExtension("matchBrackets", function() {matchBrackets(this, true);});
-  CodeMirror.defineExtension("findMatchingBracket", function(pos, strict, config){
```

Comment:
This would be a breaking change, please leave the argument list the same or at least provide backwards-compatibility code that detects and handles the old signature.